browser-test/src/support/applicant_questions.ts

```diff
@@ -741,6 +742,52 @@ export class ApplicantQuestions {
     }
   }
 
+  async expectCommonIntakeConfirmationPageNorthStar(
+    wantUpsell: boolean,
+    wantTrustedIntermediary: boolean,
+    wantEligiblePrograms: string[],
+    northStarEnabled = true,
```

Comment:
**suggestion:** We should be able to remove this too, since this is the northstar specific version and the flag isn't being used.

Reply:
Done.